src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -209,4 +217,27 @@ case class AvroRelation(location: String, minPartitions: Int = 0)
       case other => sys.error(s"Unsupported type $other")
     }
   }
+
+  // The function below was borrowed from JSONRelation
+  override def insert(data: DataFrame, overwrite: Boolean) = {
```

Comment:
Also please add a test for this. I'm gonna go ahead and annotate each new method that should have a test.